src/components/FeedModal/FeedModal.stories.js

```diff
@@ -0,0 +1,10 @@
+import FeedModal from './FeedModal';
+
+export default {
+  title: 'FeedModal',
```

Comment:
마찬가지로 title 수정해 주시면 될 것 같습니다!

```suggestion
  title: 'components/FeedModal',
```

Reply:
[e49429e](https://github.com/prgrms-web-devcourse/Team_KPPL_STUTI_FE/pull/28/commits/e49429e213368febb09bd6f496f2e0b98dd37ea9)
변경완료 했습니다. !